evolver/base.py

```diff
@@ -312,7 +312,7 @@ def init_descriptors(self, **non_config_kwargs):
         init_and_set_vars_from_descriptors(self, **non_config_kwargs)
 
     def _setup_logger(self):
-        self.logger = logging.getLogger(self.name)
+        self.logger = logging.getLogger(f"{__package__}.{self.name}")
```

Comment:
[comment] It might be slightly more robust against future changes if we added a setting for this rather than relying on the convention of it always being ``__package__``.
E.g.:
```python
# evolver.settings
class Settings(...):
  ROOT_LOGGER_NAME: str = __package__
```